core/rio/languages/src/main/java/org/eclipse/rdf4j/rio/languages/BCP47LanguageHandler.java

```diff
@@ -0,0 +1,101 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Eclipse RDF4J contributors, Aduna, and others.
```

Comment:
My remark was under the assumption that this is a new file (and not code that was already present in Sesame). The copyright line that says "Eclipse RDF4J contributors, Aduna, and others" is _only_ used for content that was already present in Sesame. For everything new, we use copyright header that is present in the current template (and you can also quickly copy-paste it from the [code formatting section in the contributor guidelines](https://github.com/eclipse/rdf4j/blob/master/.github/CONTRIBUTING.md#code-formatting), if needed):

```
/*******************************************************************************
 * Copyright (c) 2016 Eclipse RDF4J contributors.
 * All rights reserved. This program and the accompanying materials
 * are made available under the terms of the Eclipse Distribution License v1.0
 * which accompanies this distribution, and is available at
 * http://www.eclipse.org/org/documents/edl-v10.php.
 *******************************************************************************/
```

As you can see the difference is really minor.